### PR TITLE
Feature/rds

### PR DESCRIPTION
**Pull request by DhruvJWork**
DDL standalone script updated

### CODE REVIEW

File: lambda_function/requirements.txt [hunk 1]
```diff
@@ -1 +1,6 @@
-psycopg2-binary==2.8.6
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (jennifer-gft) [inline lambda_function/requirements.txt]: these changes should be reverted, only lambda libs should be added here

File: scripts/create_pkg.sh [hunk 2]
```diff
@@ -1,18 +1,44 @@
 #!/bin/bash
```
Comment (jennifer-gft) [inline scripts/create_pkg.sh]: Again the changes should be reverted, we don't need this for local lambda

File: lambda_function/package/populate.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
+import os
```
Comment (jennifer-gft) [inline lambda_function/package/populate.py]: Please create a folder called 'post-deployment-scripts' (just like scripts folder) and move this python there.

**Review by jennifer-gft — CHANGES_REQUESTED** [review]

File: lambda_function/requirements.txt (hunk 1 above)
Comment (DhruvJWork) [reply lambda_function/requirements.txt]: resolved

File: scripts/create_pkg.sh (hunk 2 above)
Comment (DhruvJWork) [reply scripts/create_pkg.sh]: resolved

File: lambda_function/package/populate.py (hunk 3 above)
Comment (DhruvJWork) [reply lambda_function/package/populate.py]: resolved

**Review by jennifer-gft — APPROVED** [review]

**Pull request merged**